module/userProjects/helper.js

```diff
@@ -384,16 +383,15 @@ module.exports = class UserProjectsHelper {
                 }
                 
                 //  push project details to kafka
-                // await kafkaProducersHelper.pushProjectToKafka(projectUpdated);
+                await kafkaProducersHelper.pushProjectToKafka(projectUpdated);
             
                 return resolve({
                     success: true,
                     message: CONSTANTS.apiResponses.USER_PROJECT_UPDATED,
                     data : {
-                        // programId : 
-                        // projectUpdated.programInformation && projectUpdated.programInformation._id ?
-                        // projectUpdated.programInformation._id : "",
-                        projectId : projectUpdated._id,
```

Comment:
@Prajwal17Tunerlabs why projectId is removed

Reply:
because the code is changed accordingly